### PR TITLE
Feature/corporations

### PR DESCRIPTION
**Pull request by Donatello-Carboni**

### CODE REVIEW

File: Documentation/Images/Temp/Deleted/funnyvideo/Deleted/funnyvideo/Deleted/funnyvideo/funnyvideo.mp4
Comment (JordanTimberlake) [inline Documentation/Images/Temp/Deleted/funnyvideo/Deleted/funnyvideo/Deleted/funnyvideo/funnyvideo.mp4:1]: Happy with this

**Review by JordanTimberlake — APPROVED** [review]
Happy that the video is back

File: Desktop/src/components/SignUpForm.svelte
Comment (keamothapo) [inline Desktop/src/components/SignUpForm.svelte:1]: please remove the comment on line 27

File: Desktop/src/electron.js
Comment (keamothapo) [inline Desktop/src/electron.js:1]: remove commented out code please

**Review by keamothapo — APPROVED** [review]

File: Desktop/src/electron.js
Comment (JordanTimberlake) [reply Desktop/src/electron.js:1]: Not those lines - Those are okay for development purposes

**Pull request merged**